logging/kit/ctxkit/context.go

```diff
@@ -37,9 +37,7 @@ func Extract(ctx context.Context) log.Logger {
 	}
 	// Add grpc_ctxtags tags metadata until now.
 	fields := TagsToFields(ctx)
-	// Add fields added until now.
-	fields = append(fields, l.fields...)
-	return log.With(l.logger, fields...)
+	return log.With(l.logger, append(fields, l.fields...))
```

Comment:
This seems to break the tests. `append(fields, l.fields...)...` should be the correct change.